_build/templates/default/sass/index.scss

```diff
@@ -1,3 +1,4 @@
+@import "../node_modules/normalize-scss/fork-versions/default/normalize";
```

Comment:
according to the [docs](https://github.com/JohnAlbin/normalize-scss) this should be used (Approach 2 – no mixin): 

`@import "[path to]/normalize-scss/sass/normalize/import-now"`

Reply:
The fork-versions are meant to be edited after copying.

Reply:
@Jako Thanks